assets/section-featured-product.css

```diff
@@ -0,0 +1,76 @@
+.featured-product {
+  margin-bottom: 5rem;
+}
+
+.featured-product .product__media-list {
+  width: 100%;
+  margin: 0;
+  padding-bottom: 0;
+}
+
+.featured-product .product__media-item {
+  padding-left: 0;
+  width: 100%;
+}
+
+.featured-product .product__media-item:not(:first-child) {
+  display: none;
+}
+
+.featured-product-section .background-secondary {
+  padding-top: 5rem;
+  padding-bottom: 0.1rem;
+}
+
+.background-secondary .featured-product {
+  background: rgb(var(--color-background));
+  padding: 2rem 2.5rem;
+}
+
+.product__view-details {
+  display: block;
+}
+
+.product__view-details .icon {
+  width: 1.2rem;
+  margin-left: 1.2rem;
+}
+
+.featured-product .share-button:nth-last-child(2) {
+  display: inline-flex;
+}
+
+.share-button + .product__view-details {
+  display: inline-flex;
+  float: right;
```

Comment:
I think we'll need to add a "clearfix" using `:after` so we don't have end up with floating issue.